### DISCUSSION TITLE
阿里云DLF

### DISCUSSION THREAD

**Discussion by zfno111** [apache/doris#27599] [A - General / Q&A]
目前使用doris(catalog) 连接阿里云的DLF  默认连接上去的是目录A
如何实现连接目录B呢？
如下图所示 我有三个目录 每个目录都相当于一个hadoop集群 里面存放了库表。
看了doris的官网 没有找到有参数可以控制连接的目录。 阿里云官网支持CatalogId来区分。
![image](https://github.com/apache/doris/assets/32893310/d5642798-6090-4963-a949-973edc7e3c7e)

**Comment by ghost** [comment]
dlf.uid应该是这个参数吧
